doc/testing_tools/testing_floating_points.qbk

```diff
@@ -83,23 +83,29 @@ BOOST_AUTO_TEST_CASE(test, * utf::tolerance(0.02))
 }
 ``` 
 
+Finally, note that comparisons for tolerance are also applied to `operator<` with semantics 'less by more than some tolerance`,
+and other relational operators. Also, the tolerance-based comparisons are involved when a more complicated expression tree is
+processed within the assertion body:
 
+[bt_example tolerance_03..tolerance applied in more complex expressions]  
 
-[section:floating_points_comparison_theory Theory]
+[/############################################################################################]
 
-In most cases it is unreasonable to use an `operator==(...)` for a floating-point values equality check.
-The simple, absolute value comparison based, solution for a floating-point values `u`,
-`v` and a tolerance `epsilon`:
+[section:floating_points_comparison_theory Tolerance-based algorithms]
+
+
+The following is the most obvious way to compare two floating-point values `u` and `v` for being close at a given absolute tolerance `epsilon`:
 
 [#equ1]
 ``
    abs(u - v) <= epsilon; // (1)
 ``
 
+However, in many circumstances, this is not what we want. A 1% tolerable difference between two huge numbers and a 1% tolerable difference
+between two tiny numbers make two different absolute tolerances. See [link Squassabia] for examples.
 
-does not produce expected results in many circumstances - specifically for very small or very big values (see
-[link Squassabia] for examples). The __UTF__ implements floating-point comparison algorithm that is
-based on the more confident solution first presented in [link KnuthII Knuth]:
+We do not want to apply the same absolute tolerance for huge and tiny numbers. Instead, we would like to scale the `epsilon` with `u` and `v`.
```

Comment:
Equ4&5 are doing exactly the opposite: they are removing the scale from u and v rather than scaling epsilon